includes/functions.php

```diff
@@ -76,3 +76,130 @@ function get_stories_theme_support() {
 
 	return $theme_support;
 }
+
```

Comment:
Really feels like all these functions should live in a class / trait. What is the benefit of them being in functions which is for public functions.